Possible docs typo (#23)
CliMA/ClimaDiagnostics.jl: docs/src/user_guide.md

- two `schedule` functions that determine when the variable should be computed
  and output (`compute_schedule_func` and `output_schedule_func`). We have two
  separate entries one for compute and one for output because we might want to
  control them separately. For example, we might want to take the average of
  something every 10 steps, and output it the average every 100 iterations.
  `schedule` functions are powerful, so there is an entire section dedicated to
  them below. `compute_schedule_func` and `output_schedule_func` are likely

>   going to the same (? same what ? output ?) unless there are temporal reductions.

Reviewer: ```suggestion
  going to be the same unless there are temporal reductions.
```

end
```

> Strictly speaking, schedules do not have to (? be ?) functions, but callable objects. For

Reviewer: ```suggestion
Strictly speaking, schedules do not have to be functions, but callable objects. For
```
